packages/components/forms/src/select/Select.styles.ts

```diff
@@ -0,0 +1,87 @@
+import tokens from '@contentful/f36-tokens';
+import { css, cx } from 'emotion';
+import { SelectWidth } from './types';
+
+function getStylesForSize(props: { width?: SelectWidth }) {
+  switch (props.width) {
+    case 'auto':
+      return css({ width: 'auto' });
+    case 'small':
+      return css({ width: '120px' });
+    case 'medium':
+      return css({ width: '240px' });
+    case 'large':
+      return css({ width: '420px' });
+    case 'full':
+    default:
+      return css({ width: '100%' });
+  }
+}
+
+export function getSelectStyles(props: {
+  isInvalid?: boolean;
+  isDisabled?: boolean;
+  width?: SelectWidth;
+}) {
+  const select = css({
+    width: '100%',
+    display: 'block',
+    appearance: 'none',
+    padding: '10px',
```

Comment:
I guess it should be the same like for inputs: `padding: `10px ${tokens.spacingS}`